README.md

```diff
@@ -286,15 +286,58 @@ curl --header "Content-Type: application/json" \
 You should receive a json response matching something like the following:
 ```
 {
-	"caseId": {CASE_ID},
-	"status": "PASS",
-	"message": ...
+    "caseId": {CASE_ID},
+    "status": "PASS",
+    "message": ...
 }
 ```
 
+### Haskell

```

Comment:
Can you move the Haskell section under the Python section instead of after the walkthrough?

Reply:
Addressed in [643e4e8](https://github.com/puffproject/test-runner/pull/20/commits/643e4e8304b9c8f56e37ac4289373612e888a70c)